pkgs/by-name/ar/arduino-create-agent/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+  go-task,
+  darwin,
+}:
+
+buildGoModule rec {
+  pname = "arduino-create-agent";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "arduino";
+    repo = "arduino-create-agent";
+    rev = "${version}";
+    hash = "sha256-TWyjF/2F3ub+sGFOTWc3kv2w6SRrvDaBSztOki32oxc=";
+  };
+
+  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin (
+    with darwin.apple_sdk.frameworks;
+    [
+      AppKit
+      Cocoa
+    ]
+  );
```

Comment:
I think you can omit these, thanks to the recent SDK updates.

Reply:
Alright, thanks